### PR TITLE
get private conversation

### PR DESCRIPTION
**Pull request by alekott**

### CODE REVIEW

**Review by e-stoj — APPROVED** [review]

File: src/main/java/com/example/chat/groupchatbackend/Conversation.java [hunk 1]
```diff
@@ -71,4 +71,8 @@ public void setConversationType(ConversationType conversationType) {
     public void setUsers(List<User> users) {
         this.users = users;
     }
+
+    public boolean checkUser(User user){
```
Comment (e-stoj) [inline src/main/java/com/example/chat/groupchatbackend/Conversation.java]: non desriptive name

File: src/main/java/com/example/chat/groupchatbackend/Conversation.java [hunk 2]
```diff
@@ -71,4 +71,13 @@ public void setConversationType(ConversationType conversationType) {
     public void setUsers(List<User> users) {
         this.users = users;
     }
+
+    public boolean checkUserPresenceInConversation(User user){
+        for (User currentUser : getUsers()){
```
Comment (michal-marciniec) [inline src/main/java/com/example/chat/groupchatbackend/Conversation.java]: replace for -> stream and use `anyMatch` method

File: src/main/java/com/example/chat/groupchatbackend/controllers/MessagesController.java [hunk 3]
```diff
@@ -83,4 +84,22 @@ private Message getMessageById(int messageId) {
         return messagesRepository.findById(messageId).orElseThrow(() -> new RuntimeException("message doesn't exist"));
     }
 
+    @GetMapping("/messages/private/{conversationId}")
+    public List <Message> getPrivateConversationMessagesWithUser(@PathVariable int conversationId, @RequestParam(required = false) Long timestamp) {
```
Comment (michal-marciniec) [inline src/main/java/com/example/chat/groupchatbackend/controllers/MessagesController.java:88]: Hmmm.... we will need to refactor this one later 😛

**Review by michal-marciniec — CHANGES_REQUESTED** [review]

**Review by michal-marciniec — APPROVED** [review]
Please merge fresh master. then ready to go

**Pull request merged**